tsconfig.json

```diff
@@ -0,0 +1,32 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "commonjs",
+    "jsx": "preserve",
+    "moduleResolution": "node",
+    "importHelpers": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "noImplicitAny": true,
+    "checkJs": true,
+    "allowJs": true,
+    "outDir": "dist",
+    "rootDir": ".",
+    "strict": true,
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "declaration": true,
+    "emitDeclarationOnly": true,
+    "isolatedModules": true,
+    "strictNullChecks": true,
+    "resolveJsonModule": true,
+    "sourceMap": true,
+    "baseUrl": ".",
+    "paths": {
+      "@commons/*": ["commons/*"],
+      "@application/*": ["core/application/*"],
+    },
+  },
+  "include": ["commons", "core", "clients", ".eslintrc.js", "babel.config.js", "jest.config.ts"],
+  "exclude": ["node_modules/**/*", "dist", ".idea", ".vscode", ".github", "coverage"],
```

Comment:
recommended to have a seperate file for this.

IMHO, we can have a `tsconfig.paths.json` file which will have the include exclude attr and later we can extend the path in the tsconfig.json

Reply:
i believe we can always extend the root `ts-config.json` in other services to include/exclude paths. making files separate will require us to do so in every service. i think rather than making it a mandatory thing, i feel we can keep it optional by adding all the necessary folders, that we believe will contain js/ts codes, and include/exclude more when necessary.

or i may have misunderstood your point, in which case, please pardon my ignorance. if you disagree with the above view, could you please suggest how that would improve in terms of performance compared to the current implementation?

Reply:
LGTM